odl/solvers/util/callback.py

```diff
@@ -946,6 +946,65 @@ def __repr__(self):
         return '{}({})'.format(self.__class__.__name__, inner_str)
 
 
+class CallbackProgressBar(Callback):
+
+    """Callback for displaying a progress bar.
+
+    This callback requires the ``tqdm`` package.
+    """
+
+    def __init__(self, niter, step=1, **kwargs):
+        """Initialize a new instance.
+
+        Parameters
+        ----------
+        niter : positive int, optional
+            Total number of iterations.
+        step : positive int, optional
+            Number of iterations between output. Default: 1
+        kwargs
+            Further parameters passed to ``tqdm.tqdm``.
+
+        Examples
+        --------
+        Create progress-bar with 10 iterations
+
+        >>> callback = CallbackProgressBar(niter=10)
```

Comment:
Fails Travis who doesn't have `tqdm`. Not really sure how to circumvent this, though.

Reply:
I'll simply remove the doctests for now, use is frankly obvious